runhouse/resources/hardware/cluster.py

```diff
@@ -919,6 +948,11 @@ def restart_server(
                 )
                 cluster_cert_path = f"{base_caddy_dir}/{self.cert_config.CERT_NAME}"
 
+        if disable_observability is None:
```

Comment:
oh i see, so here we would actually need the setter to update the property? so i guess we really should have both

Reply:
Yeah, but maybe we could just do something like `self.disable_observability = <DEFAULT disable_observability value>` instead of having the setter? Because the setting the disable_observability for true to false complicates our managing of this (prob will need to restart the server..)